pkg/providers/vsphere/validator.go

```diff
@@ -207,6 +210,19 @@ func (v *Validator) validateControlPlaneIp(ip string) error {
 	return nil
 }
 
+func (v *Validator) validateWorkerNodeGroupName(spec *spec, workerNodeGroupConfigs []anywherev1.WorkerNodeGroupConfiguration) error {
+	if workerNodeGroupConfigs[0].Name != "md-0" {
+		logger.Info("First worker node group must be named md-0. Will default name to md-0.")
```

Comment:
This is a requirement? Why is that?

Reply:
I updated it to only default to this name when there's only 1 worker node group & the name isnt defined. That way existing clusters can still be supported with these changes

Reply:
Oh so we have a separate requirement of name if there are multiple defined? We could even have it as the default of cluster spec from now right?